change/@microsoft-teams-js-521260e3-aa09-4bcd-a9c9-3f0cabcc51a4.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Update title as optional in stageViewParams",
```

```suggestion
  "comment": "Made title optional when calling `stageView.open`",
```

These messages are intended for external developers who are building apps, so we want the message to accurately and succinctly capture what the change means for those developers. Does this suggestion sound correct?

---

Yes, thanks!